go.mod

```diff
@@ -3,21 +3,37 @@ module github.com/carlpett/terraform-provider-sops
 go 1.12
```

Comment:
Shouldn't this get bumped with the ci upgrade?

Reply:
I would have expected so... Not sure why it isn't. I've run all the build/test/etc steps with go 1.13, which has bumped it on other projects, but it doesn't seem to touch it :man_shrugging: 
AFAIK it doesn't have much effect, but I'll fix the file manually.